src/store/reducers/dform/initial/styles.json

```diff
@@ -479,7 +479,7 @@
               },
               "units": [
                 "kg",
-                "lbm"
```

Comment:
Lbm was to stand for lbmass, but pounds in English countries are almost always abbreviated as 'lb'

Reply:
😮 Thanks!